nanoFirmwareFlasher/EspTool.cs

```diff
@@ -578,6 +578,24 @@ internal ExitCodes EraseFlashSegment(uint startAddress, uint length)
             {
                 appName = "esptool";
                 appDir = Path.Combine(Program.ExecutingPath, "esptool", "esptoolLinux");
+                Process espToolExex = new Process();
+                // Making sure the esptool is executable
+                espToolExex.StartInfo = new ProcessStartInfo("chmod +x", Path.Combine(appDir, appName))
```

Comment:
Can't the permissions be set once and for all instead of doing it every time process is fired up?

Reply:
That's then safer way